README.md

```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
-    "foobar": "Foo/bar#^2.0.0"
-  }
-}
-```
-
-`bower_components-polymer-2`:
 
-```json
-{
-  "name": "example",
-  "dependencies": {
-    "polymer": "Polymer/polymer#^2.0.0",
-    "foobar": "Foo/bar#^2.0.0"
-  }
-}
-```
-
-Commands like `install`, `serve` and `test` recognize that variants are
-installed to offer additional support.
-
-`polymer install --variants` installs dependency variants into the proper
-folders.
+## Command Overview
 
-`polymer serve` will start up a development server for
-each variant on its own port so that you can easily switch between variants,
-viewing their demos and tests.
+### `polymer help [COMMAND]`
 
-`polymer test` will run the test suite once for each variant, ensuring that you
-have test coverage against supported configurations.
+Run `polymer help` to get a helpful list of supported commands. Pass it a command name (ex: `polymer help serve`) to get detailed information about that command and the options it supports.
 
-## Commands
 
-### help
+### `polymer init [TEMPLATE]`
 
-Displays help on commands and options:
+Initializes a Polymer project from one of several templates. Pre-bundled templates range from just bare-bones to fully featured applications like the [Polymer Starter Kit](https://github.com/PolymerElements/polymer-starter-kit).
 
-    $ polymer help
+You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for a template you'd like to use. Then install it and the CLI will pick it up automatically.
 
-### init
+Run `polymer init` to choose a template from a list of all installed templates. Or if you know the template name before hand, you can provide it as a command argument.
 
-Initializes a Polymer project from one of several templates.
 
-Choose a template from a menu:
+### `polymer install [--variants]`
 
-    $ polymer init
+Install your dependencies, similar to running `bower install`.
 
-Create a new project from the 'element' template:
+If the `--variants` option is provided, the command will also search your project's `bower.json` for  a `"variants"` property and install any [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) listed there as well as well. This is useful if you need to test your elements against multiple versions of Polymer and/or sets of dependencies.
 
-    $ polymer init element
 
-You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for the full list.
+### `polymer serve [options...]`
 
-### install
+Start a development server designed for serving Polymer & Web Component projects. Applications are served as-is, while elements are served from a special route where it can properly reference its dependencies.
 
-Installs Bower dependencies, optionally installing multiple variants.
+Run `polymer help serve` for the full list of available options.
```

Comment:
See comment above re: `[...options]`. This lets us point out that there are a collection of important features not listed here which are explained in more detail by `polymer help serve`.

Maybe there's a more clear way to signify this? Any ideas?